firmware/main.c

```diff
@@ -1,18 +1,36 @@
-/* 
- * Project: Micronucleus -  v2.04
```

Comment:
I think we should turn this into 2.5, otherwise things will be very confusing

Reply:
OK